atest/testdata/standard_libraries/builtin/verify.robot

```diff
@@ -68,6 +68,11 @@ Should Not Be Equal
     ${STR1}    ${INT1}
     ${STR1}    1
 
+Should Not Be Equal With Case Insensitivity
+    [Template]  Should Not Be Equal
+    test value      TEST VALUE1      ignore_case=True
+    HYVÄÄ YÖTÄ      hyvää yötä1     ignore_case=True
```

Comment:
It would be good to test that e.g. `foo == FOO` fails when `ignore_case=True`. The current values are unequal also with `ignore_case=False`.

There's also different number of spaces before `ignore_case` above.